server/config/persist_options.go

```diff
@@ -736,6 +744,26 @@ func (o *PersistOptions) getTTLUintOr(key string, defaultValue uint64) uint64 {
 	return defaultValue
 }
 
+func (o *PersistOptions) getTTLBool(key string) (result bool, contains bool, err error) {
```

Comment:
How about making it consistent with `enable-location-replacement`?

Reply:
we can change it in later pr.